src/less/infotip/infotip.less

```diff
@@ -112,6 +112,7 @@ span.infotip__heading {
 .infotip .icon-btn {
     flex-shrink: 0; // todo: Should move to icon-btn in next major
     height: 20px;
+    line-height: 16px;
```

Comment:
This doesn't look accurate. Visually, it's still not centered. Since this is a graphical element and has no visible text, you probably need to set `line-height: 0`.
<img width="415" alt="image" src="https://github.com/eBay/skin/assets/1675667/42a2e1bc-2c38-4026-8db3-ad2c55c7be19">

Reply:
I can't verify any negative implications downstream since I can't current run storybook (errors out).

Reply:
Actually, doing `display: inline-flex;` creates perfect centering without `line-height` pixel specification. If that doesn't create negative downstream implications, it might be preferable. 
<img width="734" alt="image" src="https://github.com/eBay/skin/assets/1675667/d9c9c883-d9e3-46aa-ae3c-024512685599">

Reply:
I had tried that but on paragraph it was off. 
Where did you put the inline flex?

Reply:
Okay, updated it to flex.